test/cases/payments/p2p.test.ts

```diff
@@ -0,0 +1,149 @@
+import p2p from '../../../topologies/p2p.json';
+import { LuminoTestEnvironment } from '../../../src/types/lumino-test-environment';
+import { LuminoNode } from '../../../src/types/node';
+import { Dictionary } from '../../../src/util/collection';
+import setupTestEnvironment from '../../../src';
+import { sleep, verifyChannel } from '../../utils';
+import { tokenAddresses, toWei } from '../../../src/util/token';
+import {
+    SETUP_TIMEOUT,
+    TEARDOWN_TIMEOUT,
+    ChannelState,
+    State,
+    TEST_TIMEOUT,
+} from '../../common';
+
+interface paymentParams {
```

Comment:
this one is repeated from `mediated`, but i can't decide were to place it honestly. maybe a common file inside the payments tc dir?

Reply:
Maybe we could add a test/types dir, or something like that. I'm not very sure either

Reply:
yes, but thinks like `depositParams` are really only used in the deposit test, so i believe that one should stay inside the test and not in a types file. unless of course we add it to the sdk code, then it would probably be best to move it

Reply:
I'd then leave it this way, and update that when we tackle the corresponding issue